subsys/bluetooth/host/Kconfig

```diff
@@ -492,6 +492,18 @@ config BT_BACKGROUND_SCAN_WINDOW
 	int "Scan window used for background scanning in 0.625 ms units"
 	default 18
 	range 4 16384
+
+if BT_EXT_ADV
+config BT_EXT_SCAN_BUF_SIZE
+	int "Maximum advertisement report size"
+	range 1 1650
+	default 255
```

Comment:
I think perhaps a default of 229 (maximum size of a single extended advertising data value) is better.

Reply:
I don't have a strong opinion on the default value, but I believe the value 255 scales better as it allows us to use the same default value for the extended scan buffer and the synchronized receive buffer. Periodic Advertising Reports have a maximum data length of 247 bytes.

Reply:
> I believe the value 255 scales better as it allows us to use the same default value for the extended scan buffer and the synchronized receive buffer.

I don't see the value of synchronizing the default value of those two, especially since the extended scan buffer can be larger than 247 (there's no fragmentation for periodic advertisements), and they are used for significant different purposes.

Reply:
There is fragmentation in Periodic Advertising Reports. See the parameter named `Data_Status` in Core_v5.3, Vol 4, Part, E, Section 7.7.65.15.

Reply:
You are absolutely right - I remembered that it had it but clearly I missed that field when looking it up earlier today :)